docs/explanation/logical-axiomatization.md

```diff
@@ -0,0 +1,101 @@
+- `chandelier pvalb GABAergic cortical interneuron` has the subclass axioms `'has characteristic' some 'chandelier cell morphology'` and `interneuron`
+- `chandelier pvalb GABAergic cortical interneuron` is therefore a subclass of `chandelier cell`
+
+Equivalent class axioms classification can be very powerful as it takes into consideration complexed layers of axioms. 
+
+For example: 
+- `primary motor cortex pyramidal cell` has the equivalent class axiom `'pyramidal neuron' and ('has soma location' some 'primary motor cortex')`. 
+- `Betz cells` have the axioms `'has characteristic' some 'standard pyramidal morphology'` and `'has soma location' some 'primary motor cortex layer 5'`
+- `Betz cells` are inferred to be `primary motor cortex pyramidal cell` through the following chain (you can see this in protege by pressing the ? button on inferred class):
+
+![](../images/discussions/logical-axiomatization/betz-pyramidal.png) 
+
+The ability of the reasoner to infer complex classes helps in classifications that might have been missed if done manually. However, when creating an equivalent class axiom, you must be sure that it is not overly constrictive (in which case, classes that should be classified under it gets missed) nor too loose (in which case, classes will get wrongly classifed under it)
+
+Example of both overly constrictive and overly loose equivalent class axiom:
+
+`neuron equivalent to cell and (part_of some 'central nervous system')`
+
+- This is overly constrictive as there are neurons outside the central nervous system (eg peripheral neurons).
+- This is also too loose as there are cells in the central nervous system that are not neurons (eg glial cells).
+
+In such cases, sometimes not having an equivalent class axioms is better (like in the case of neuron), and asserting is the best way to classify a child. 
+
+## Style guide
+
+Each ontology has certain styles and conventions in how they axiomatize. This style guide is specific to OBO ontologies. We will also give reasons as to why we choose to axiomatize in the way we do. However, be aware of your local ontology's practices. 
+
+### Respect the ontology style
+
+As a preamble (and to avoid over prescribing style), it is important for us to note that ontologies have specific styles to things like what relation to use, and should be respected. This is usually important for their use cases. For example the cell ontology has a [guide to what relations to use](https://obophenotype.github.io/cell-ontology/relations_guide/). An example of an agreement in the community is that while anatomical locations of cells are recorded using `part of`, neurons should be recorded with `has soma location`. This is to accommodate for the fact that many neurons have long reaching synapses that cover multiple anatomical location making them tricky to axiomatize using `part of`. 
```

Comment:
Suggested:
It is important to note that ontologies have specific axiomatization styles and may apply to, for example, selecting a preferred relation. This is usually reflects their use cases. For example, the Cell Ontology has a [guide for what relations to use](https://obophenotype.github.io/cell-ontology/relations_guide/). An example of an agreement in the community is that while anatomical locations of cells are recorded using `part of`, neurons should be recorded with `has soma location`. This is to accommodate for the fact that many neurons have long reaching axons that cover multiple anatomical locations making them difficult to axiomatize using `part of`.